netbox/dcim/models/racks.py

```diff
@@ -542,9 +542,9 @@ def get_utilization(self):
                 available_units.remove(u)
 
         occupied_unit_count = self.u_height - len(available_units)
-        percentage = int(float(occupied_unit_count) / self.u_height * 100)
 
-        return percentage
+        # Return the numerator and denominator as percentage is to be calculated later where needed
+        return (occupied_unit_count, self.u_height)
```

Comment:
From a readability and usability perspective this seems like it'd be a great use case for `collections.namedtuple`.